data/CMakeLists.txt

```diff
@@ -8,5 +8,6 @@ if(USE_XMLLINT AND XMLLINT_FOUND)
 endif()
 
 if(WITH_PUGIXML)
+  FILE(COPY cameras.xml showcameras.xsl DESTINATION ${DARKTABLE_DATADIR}/rawspeed)
```

Comment:
What's `DARKTABLE_DATADIR`?
If rs is being built outside of darktable, this now tries to copy a file into root file system directory.

Reply:
yeah, sorry, my bad. I have to work on that.

Reply:
I will remove that PR because the lib (for dt) is built in (CMAKE_CURRENT_BINARY_DIR) ./build/lib/darktable/rawspeed/data/rawspeed/cameras.xml but darktable expects to find it in build/share/darktable/rawspeed/cameras.xml. So the copying needs to take place outside the CMakeLists.txt scripts from rawspeed.